app/test_main.py

```diff
@@ -1,3 +1,36 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, result", [
+        pytest.param(
+            0, 0, [0, 0], id="test_should_return_correct_result_1"
+        ),
+        pytest.param(
+            14, 14, [0, 0], id="test_should_return_correct_result_2"
+        ),
+        pytest.param(
+            15, 15, [1, 1], id="test_should_return_correct_result_3"
+        ),
+        pytest.param(
+            23, 23, [1, 1], id="test_should_return_correct_result_4"
+        ),
+        pytest.param(
+            24, 24, [2, 2], id="test_should_return_correct_result_5"
+        ),
+        pytest.param(
+            27, 27, [2, 2], id="test_should_return_correct_result_6"
+        ),
+        pytest.param(
+            28, 28, [3, 2], id="test_should_return_correct_result_7"
```

Comment:
The expected result for `get_human_age(28, 28)` should be `[3, 3]` instead of `[3, 2]`. Please update this test case accordingly.